modules/ask_va_api/spec/services/dynamics/service_spec.rb

```diff
@@ -0,0 +1,173 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Dynamics::Service do
```

Comment:
add a test for the prepare_payload method that verifies an error gets raised if the method passed in is not get, post, patch, or put

Reply:
same as above